lookupy/tests.py

```diff
@@ -108,6 +108,7 @@ def test_lookup():
     # in          -- works for strings and lists, else raises error
     assert lookup('request__url__in', ['http://example.com',
                                        'http://blog.example.com'], entry1)
+    assert lookup('request__url__not_in', ['http://example.io'], entry1)
```

Comment:
Minor suggestion: Notice that the assertions are grouped by the operator (comments like `# in ...` etc. that are separating various assertions. It'd be good if you can add a separate section `# not in ...`  following the same convention.